docs/Features/XMLFeatures.md

```diff
@@ -0,0 +1,125 @@
+# XML Features
+
+[vscode-xml](https://github.com/redhat-developer/vscode-xml) provides a number of notable features, with demos below.
+
+## XML Tag Auto Close
+
+XML tag `auto-close` is supported:
+
+Using `/` in an opening tag will auto close the tag.
+
+![Auto Close Tag Slash](../images/Features/AutoCloseTagSlashXML.gif)
+
+## Linked Editing
+
+Linked editing is supported, allowing for simultaneous changes an opening and closing tag pair.
+
```

Comment:
To benefit with this feature, you must activate it in your `settings.json` with `"editor.linkedEditing": true`